HW_7/HW_7.2_Custom_filter_function.py

```diff
@@ -0,0 +1,43 @@
+"""
+2. Implement your realization of the function filter
+Note: tasks 2-3 must have the following signature def function_name(callback, sequence). Add annotations yourself.
+Functions must be able to work with integers, floats, and strings as elements of sequences.
+"""
+
+
+def custom_filter(callback, sequence):
+    """
+    Function custom_filter is used to filter elements based on specific conditions. it returns a sequence from those 
+    elements of iterable for which function returns True. 
+    :param callback: function with specific filter rules that
+    returns True only for those elements that have to be in filtered result 
+    :param sequence: list, set, tuple with allowed int, float, str values
+    :return: object with int, float, str values
+    """
```

Comment:
it is better to use annotations than doc strings.

Reply:
Added annotations in addition to doc string (want to keep it with more explicit explanation).